### PR TITLE
infra: add javascript as a language with builder img

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Ref: https://github.com/google/oss-fuzz/issues/11103

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
Also adjusts some formatting which showed up when running `presubmit.py`

**Review by Alan32Liu — APPROVED** [review]

**Review by jonathanmetzman — APPROVED** [review]
LGTM with nits

File: infra/helper.py [hunk 1]
```diff
@@ -256,11 +258,13 @@ def get_parser():  # pylint: disable=too-many-statements,too-many-locals
   generate_parser = subparsers.add_parser(
       'generate', help='Generate files for new project.')
   generate_parser.add_argument('project')
-  generate_parser.add_argument(
-      '--language',
-      default=constants.DEFAULT_LANGUAGE,
-      choices=['c', 'c++', 'rust', 'go', 'jvm', 'swift', 'python', 'javascript'],
-      help='Project language.')
+  generate_parser.add_argument('--language',
```
[the remark this answers is not in the document]
Comment (DavidKorczynski) [reply infra/helper.py:268]: Updated, but LANGUAGES_WITH_BUILDER_IMAGES to be a mapping rather than a list, wdyt?

File: infra/helper.py [hunk 2]
```diff
@@ -77,7 +77,16 @@
 # Regex to match special chars in project name.
 SPECIAL_CHARS_REGEX = re.compile('[^a-zA-Z0-9_-]')
 
-LANGUAGES_WITH_BUILDER_IMAGES = {'go', 'jvm', 'python', 'rust', 'swift'}
+LANGUAGE_TO_BASE_BUILDER_IMAGE = {
+    'c': 'base-builder',
+    'c++': 'base-builder',
+    'go': 'base-builder-go',
+    'javascript': 'javascript',
```
Comment (oliverchang) [inline infra/helper.py:84]: base-builder-javascript?

Comment (DavidKorczynski) [reply infra/helper.py]: Yes!

**Review by oliverchang — APPROVED** [review]

**Pull request merged**